### DISCUSSION TITLE
Is Buildx/BuildKit planned to be the default build daemon for Docker?

### DISCUSSION THREAD

**Discussion by nabeel-halo** [docker/buildx#1158] [Q&A] [answered]
Hi!

I'm writing some setup documentation and including the steps that after installing Docker, the build daemon should be changed to Buildx with `docker buildx install`. My question for future planning purposes (and I understand if there isn't a hard timeline for this) is, is it planned that in the future, Buildx will become the default out-of-the-box daemon for Docker? I ask because if so, I'd like to leave a note in my documentation that in the future this step may become unnecessary. But if the legacy daemon will be retained permanently for backwards-compatibility purposes, that would also be good to know. Would love to hear from the team.

Thank you!

**Comment by jedevc** [comment, marked as answer]
Yup, buildx + buildkit are going to become the default at some point.

See https://github.com/moby/moby/issues/40379 for the proposal, and https://github.com/docker/cli/pull/3314 for details of what the command line interface will change to be.

**Reply by nabeel-halo** [reply]
Thank you! Appreciate the links as well :)
